Atanda AbdulAzeez/day3.js

```diff
@@ -0,0 +1,9 @@
+const reverseString = string =>{
+    let reversedString = "";
+    for (let i = string.length; i >= 0; i--){
+       reversedString += string.charAt(i);
+    }
+    return reversedString;

```

Comment:
another option is to convert string to array of characters using `split` method then reverse the array

Nice work by the way